src/middleware/createRouterMiddleware.js

```diff
@@ -21,6 +21,9 @@ import matchRoutes from '../utils/matchRoutes';
 import cloneRoutesForKey from '../utils/cloneRoutesForKey';
 import isOnlyLocationHashChange from '../utils/isOnlyLocationHashChange';
 
+// Kangzes annotations & why
+// https://github.com/zapier/zapier/pull/29960#issuecomment-522404231
```

Comment:
@kangzeroo I'd prefer to avoid adding private links to public code like this. It's not a security problem or anything, but it's just not a useful comment for anybody that doesn't have access to our repo. Instead, I'd rather keep the necessary (public) context here.

Having said this, I think @ngryman might be deprecating this public repo, but I'm not sure what the status of that is.

Reply:
@jdeal It's pending for now as I moved to other higher priorities. So I suppose your suggestion still applies the time we migrate it to the monorepo.

Reply:
Oh shoot! I forgot this was public. I'll remove the comment in another commit (although itll still be there history) and not merge any code/delete branch.